obspy/core/inventory/inventory.py

```diff
@@ -1117,6 +1117,351 @@ def plot_response(self, min_freq, output="VEL", network="*", station="*",
+            Different start and end times are no conflict as long as
+            the times are not overlapping.
+            "standard": Stations are only kept if all relevant
+            attributes are the same.
+            "first": If station attributes are different the
+            attributes of Inventory 1 are kept.
+            "second": If station attributes are different the
+            attributes of Inventory 2 are kept.
+            "same": If Station attributes are different only not
+            conflicting ones are kept all others are set to None.
+        :type station_merge: string
+
+        :param channel_merge: Following options are available:
+            "standard": If times are not overlapping channels with the
+            same code are kept even when relevant parameters are
+            different. If times are overlapping the channel of inv_2
+            is discarded.
+            "all": All channels are kept even if the times are
+            overlapping and the parameters differ.
+        :type channel_merge: string
+
+        :param reset_created: If True the created time is set to the merge
+        time. If false it is set accordingly to inventory_merge.
+        :type reset_created: bool
+
+        :return: class:`~obspy.core.inventory.inventory.Inventory`
+        """
+
+        # list all attributes to compare networks, stations and channels
+        # with each other
+        inv_att = ["source", "sender", "module", "module_uri"]
+        net_att = ["total_number_of_stations", "description",
+                   "comments", "start_date", "end_date",
+                   "restricted_status", "alternate_code",
+                   "historical_code", "data_availability"]
+        sta_att = ["latitude", "longitude", "elevation", "site",
+                   "vault", "geology",
+                   "equipments", "operators", "creation_date",
+                   "termination_date",
+                   "total_number_of_channels", "description",
+                   "comments", "start_date", "end_date",
+                   "restricted_status", "alternate_code",
+                   "historical_code", "data_availability"]
+        cha_att = ["location_code", "latitude", "longitude",
+                   "elevation", "depth", "azimuth",
+                   "dip", "types", "external_references",
+                   "sample_rate",
+                   "sample_rate_ratio_number_samples",
+                   "sample_rate_ratio_number_seconds",
+                   "storage_format",
+                   "clock_drift_in_seconds_per_sample",
+                   "calibration_units",
+                   "calibration_units_description", "sensor",
+                   "pre_amplifier", "data_logger",
+                   "equipment", "response", "description",
+                   "comments", "start_date", "end_date",
+                   "restricted_status", "alternate_code",
+                   "historical_code", "data_availability"]
```

Comment:
This should not be necessary as the individual objects implement `__eq__` methods so they can be compared for equality that way.

Reply:
As I understand it `__eq__` compares all properties. For channels, it would not be necessary indeed. For stations and networks, I want to exclude for example the selected number, so I think the lists are necessary ?!?

Reply:
Yea that is a good point. I did not see that.